hack/jenkins/job-builder-image/Dockerfile

```diff
@@ -0,0 +1,33 @@
+# This docker image runs Jenkins Job Builder for automatic job reconciliation.
+
+FROM ubuntu:14.04
+MAINTAINER Joe Finney <spxtr@google.com>
+
+RUN mkdir /build
+WORKDIR /build
+
+# Dependencies for jenkins-job-builder
+RUN apt-get update && apt-get install -y \
+    wget \
+    git \
+    python-dev \
+    python-pip \
+    libyaml-dev \
+    python-yaml
+RUN pip install PyYAML python-jenkins
+# Required since jenkins-job-buider supports earlier versions of python than 2.7
```

Comment:
does ubuntu14.04 use python 2.6? maybe just say that explicitly in this comment to indicate that's what you mean